### ISSUE TITLE
Logging Training Information

### ISSUE THREAD

**Issue by MaximeBouton** [JuliaPOMDP/DeepQLearning.jl#11]
It would be nice to use something like this: https://github.com/oxinabox/UniversalTensorBoard.jl

To log training data and be able to use tensorboard.

**Comment by MaximeBouton** [comment]
Give a shot to: 
https://github.com/PhilipVinc/TensorBoardLogger.jl/blob/master/src/TensorBoardLogger.jl

**Comment by MaximeBouton** [comment]
implemented in #12 

It could be nice to be able to log environment specific information as well using the `info` output of the `step!` function.

**Comment by PeggyYuchunWang** [comment]
Perhaps not as important, but logging the eval_reward along with eps, avgR, loss, and Grad in the terminal during training would be cool as well.
```
48500 / 1000000 eps 0.904 |  avgR -1.004 | Loss 6.049e-01 | Grad 5.596e-01 
49000 / 1000000 eps 0.903 |  avgR -1.004 | Loss 6.760e-01 | Grad 3.692e-01 
```
